### PR TITLE
fix: add missing Migration lang item

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add missing Migration lang item

```
$ php spark migrate

CodeIgniter v4.1.6 Command Line Tool - Server Time: 2022-01-08 20:30:40 UTC-06:00

すべての新しいマイグレーションを実行しています...
Done migrations.
```

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Commands/Database/Migrate.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function run(array $params)
                 CLI::write($message);
             }
 
-            CLI::write('Done migrations.', 'green');
+            CLI::write(lang('Migrations.done'), 'green');
```
Comment (MGatner) [inline system/Commands/Database/Migrate.php:94]:
```suggestion
            CLI::write(lang('Migrations.migrated'), 'green');
```

File: system/Language/en/Migrations.php [hunk 2]
```diff
@@ -36,6 +36,7 @@
 
     'latest'            => 'Running all new migrations...',
     'generalFault'      => 'Migration failed!',
+    'done'              => 'Done migrations.',
```
Comment (MGatner) [inline system/Language/en/Migrations.php:39]:
```suggestion
    'migrated'          => 'Migrations complete.',
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Small semantic suggestion. 

@lonnieezell do we need to do anything to "request translators" when new strings get added, for the translations repo?

**Comment by sfadschm** [comment]
> Small semantic suggestion.
> 
> @lonnieezell do we need to do anything to "request translators" when new strings get added, for the translations repo?

Paul implemented static analysis checks in the translation repo that check for missing, duplicated or copied strings.

**Comment by kenjis** [comment]
> Paul implemented static analysis checks in the translation repo that check for missing, duplicated or copied strings.

But translators do not know if they run checks.

**Review by MGatner — APPROVED** [review]

**Pull request merged**